pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml

```diff
@@ -2,51 +2,51 @@
 layout: none
 theme: none
 ---
--if dashboard_user[:admin]
-  %h1
-    Ops Dashboard
-  %a{href:'/ops-dashboard#/districts'}
-    %h3
-      Manage Districts
--if dashboard_user[:admin] || have_permission?('facilitator')
-  -if !dashboard_user[:admin]
+
+.teacher-announcements
+  = view :teacher_announcements
+
+.ops-workshops
```

Comment:
diff below here is just indentation because I added the .ops-workshops wrapper div